README.md

```diff
@@ -74,17 +74,17 @@ files:
 
 ### CSV Data Files
 
-One CSV file can only store one type of vertex or edge. Vertices and edges of different schema should be stored in different files.
```

Comment:
One CSV file can only store one type of vertex or edge. Vertices and edges of different schemas must be stored in different files.